CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+help you make your first contribution. [help wanted](https://github.com/coralogix/telemetry-shippers/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) are issues
+suitable for someone who isn't a core maintainer and is good to move onto after
+your first pull request.
+
+Once you see an issue that you'd like to work on, please post a comment saying
+that you want to work on it. Something like "I want to work on this" is fine.
+
+## Ask for Help
+
+The best way to reach us with a question when contributing is to ask on the original github issue.
+
+## Pull Request Lifecycle
+
+⚠️ **Explain your pull request process**
+
+- Open a PR; if it's not finished yet, please make it a draft first
+- Reviewers will be automatically assigned based on code owners
+- Reviewers will get to the PR as soon as possible, but usually within 2 days
+
+## Development Environment Setup
+
+[TODO]
+
+⚠️ **Explain how to set up a development environment**
+
+## Sign Your Commits
+
+⚠️ **Keep either the DCO or CLA section depending on which you use**
+
+### DCO
+
+Licensing is important to open source projects. It provides some assurances that
+the software will continue to be available based under the terms that the
+author(s) desired. We require that contributors sign off on commits submitted to
+our project's repositories. The [Developer Certificate of Origin
+(DCO)](https://probot.github.io/apps/dco/) is a way to certify that you wrote and
+have the right to contribute the code you are submitting to the project.
+
+You sign-off by adding the following to your commit messages. Your sign-off must
+match the git user and email associated with the commit.
+
+```
+This is my commit message
+
+Signed-off-by: Your Name <your.name@example.com>
+```
+
+Git has a `-s` command line option to do this automatically:
+
+```
+git commit -s -m 'This is my commit message'
+```
+
+If you forgot to do this and have not yet pushed your changes to the remote
+repository, you can amend your commit with the sign-off by running
+
+```
+git commit --amend -s 
+```
+
+### CLA
+
+We require that contributors have signed our Contributor License Agreement (CLA).
+
+⚠️ **Explain how to sign the CLA**
+
+When a contributor submits their first Pull Request, the CLA Bot will step in with a friendly comment on the new pull request, kindly requesting them to sign the [Coralogix's CLA](https://cla-assistant.io/coralogix/telemetry-shippers).
+
+## Pull Request Checklist
+
+When you submit your pull request, or you push new commits to it, our automated
+systems will run some checks on your new code. We require that your pull request
+passes these checks, but we also have more criteria than just that before we can
+accept and merge it. We recommend that you check the following things locally
+before you submit your code:
+
+- CLA,
```

Comment:
We probably need to set up a required CI check to make sure we don't merge PRs without CLA signed

Reply:
Yup!
We already have the DCO bot, but not the CLA yet I think so.